### PR TITLE
DEV: Add acceptance tests for assignment notifications

### PR DESCRIPTION
**Pull request by xfalcox**
Follow up of https://github.com/discourse/discourse-assign/pull/306#issuecomment-1071953653

@tgxworld Is this what you had in mind?

Can't get it to pass tho, any ideas @CvX ?

### CODE REVIEW

File: test/javascripts/acceptance/assigned-topic-test.js [hunk 1]
```diff
@@ -123,6 +130,32 @@ acceptance("Discourse Assign | Assigned topic", function (needs) {
       "shows reassign dropdown at the bottom of the topic"
     );
   });
+
+  test("Shows assignement notification", async (assert) => {
```
Comment (CvX) [inline test/javascripts/acceptance/assigned-topic-test.js:134]:
```suggestion
  test("Shows assignment notification", async function (assert) {
```

File: test/javascripts/acceptance/assigned-topic-test.js [hunk 2]
```diff
@@ -123,6 +130,32 @@ acceptance("Discourse Assign | Assigned topic", function (needs) {
...
+    updateCurrentUser({ can_assign: true });
+
+    await visit("/u/eviltrout/notifications");
+
+    debugger;
+
+    let notification = query("section.user-content ul.notifications li.item.notification")[0];
```
Comment (CvX) [inline test/javascripts/acceptance/assigned-topic-test.js:141]: This is the showstopper you were hitting: 😃 

```suggestion
    let notification = query("section.user-content ul.notifications li.item.notification");
```

**Review by CvX — DISMISSED** [review]

**Comment by tgxworld** [comment]
@xfalcox I pushed some changes to make the tests passed. Do note that pretender consider query params when matching routes.

**Comment by xfalcox** [comment]
> @xfalcox I pushed some changes to make the tests passed. Do note that pretender consider query params when matching routes.

Ohhhh so that was it. I was using the route the app uses when navigating to it, but that wasn't it. Huge thanks!

**Review by romanrizzi — APPROVED** [review]

**Pull request merged**